copy/copy.go

```diff
@@ -709,15 +709,15 @@ func checkImageDestinationForCurrentRuntime(ctx context.Context, sys *types.Syst
 			wantedOS = sys.OSChoice
 		}
 		if wantedOS != c.OS {
-			return fmt.Errorf("Image operating system mismatch: image uses %q, expecting %q", c.OS, wantedOS)
+			logrus.Infof("Image operating system mismatch: image uses %q, expecting %q", c.OS, wantedOS)
```

Comment:
Absolutely non-blocking: We’ve had the OS check for a long time without complaints… not sure that one needs to be demoted. OTOH, it’s pretty hard to _accidentally_ refer to a Windows image on Linux computers, and the consistency with architecture is nice.